### DISCUSSION TITLE
Windows上如何实现启动时隐藏窗口

### DISCUSSION THREAD

**Discussion by XinSSS** [AlistGo/alist#1206] [Q&A] [answered]
如题, 如果想达到启动时隐藏窗口并且输出log到指定文件 , 目前能否实现?wiki里没找到

**Comment by nitsort** [comment]
> 如题, 如果想达到启动时隐藏窗口并且输出log到指定文件 , 目前能否实现?wiki里没找到

可以借助其他工具，比如：https://github.com/winsw/winsw

**Reply by anwen-anyi** [reply]
> > 如题, 如果想达到启动时隐藏窗口并且输出log到指定文件 , 目前能否实现?wiki里没找到
> 
> 可以借助其他工具，比如：https://github.com/winsw/winsw

呜呜呜...菜鸡不会用这个 只能用大佬给我的两个脚本..虽然能一直保持启动但是没法看日志就是.

**Reply by nitsort** [reply]
> > > 如题, 如果想达到启动时隐藏窗口并且输出log到指定文件 , 目前能否实现?wiki里没找到
> > 
> > 
> > 可以借助其他工具，比如：https://github.com/winsw/winsw
> 
> 呜呜呜...菜鸡不会用这个 只能用大佬给我的两个脚本..虽然能一直保持启动但是没法看日志就是.

使用可以参考这个：https://blog.csdn.net/qubernet/article/details/120290136
日志具体配置看这里：https://github.com/winsw/winsw/blob/v3/docs/logging-and-error-reporting.md

**Reply by anwen-anyi** [reply]
> > > > 如题, 如果想达到启动时隐藏窗口并且输出log到指定文件 , 目前能否实现?wiki里没找到
> > > 
> > > 
> > > 可以借助其他工具，比如：https://github.com/winsw/winsw
> > 
> > 
> > 呜呜呜...菜鸡不会用这个 只能用大佬给我的两个脚本..虽然能一直保持启动但是没法看日志就是.
> 
> 使用可以参考这个：https://blog.csdn.net/qubernet/article/details/120290136 日志具体配置看这里：https://github.com/winsw/winsw/blob/v3/docs/logging-and-error-reporting.md

么么哒~这就去试试

**Comment by Jonah-77** [comment, marked as answer]
CreateObject("WScript.Shell").Run "alist-windows-4.0-amd64.exe",0

上方命令创建vbs文件放在文件目录下可以实现隐藏窗口打开

**Reply by XinSSS** [reply]
好的，谢谢，明天我试试看

**Reply by jilefo** [reply]
https://www.52pojie.cn/thread-1713978-1-1.html 用这个帖子的工具吧。 一键自动初建，包括更新。

**Comment by abiglee** [comment]
可以参考下这篇文章，很全面
https://www.jianshu.com/p/7f31ecebda28
